agent/host_agent_test.go

```diff
@@ -561,7 +563,57 @@ var _ = Describe("Agent", func() {
 				return false
 			}, time.Second*2).Should(BeTrue())
 		})
+		It("should skip bootstrap kubeconfig flow if the ~/.byoh/config exists", func() {
```

Comment:
You have created `${HOME}/.byoh` directory but didn't clean up

Reply:
We are creating this inside container and the container gets destroyed after each test.